### PR TITLE
Document SDK calls and app creation flow

### PR DESCRIPTION
**Pull request by 0zzy-o**
* closes https://github.com/MainframeHQ/js-mainframe/issues/67
* fixes issue validating web host

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +58,93 @@ yarn build
...
+
+We recommend to build your apps using [React Native Web](https://github.com/necolas/react-native-web), you can see a sample setup in the reference app, [Onyx Stats](/applications/onyx-stats). We are currently working on a UI Kit to make it easier for developers to build Mainframe app interfaces and will also be releasing a bootstrapping script to streamline the initial setup process.
+
+Your app will need to use the [Mainframe sdk](packages/sdk) to interact with the daemon, you can view the available api's in the [sdk package](packages/sdk/README.md).
+
+Example SDK usage:
+
+```
+import sdk from '@mainframe/sdk'
+const sdk = new MainframeSDK()
```
[the remark this answers is not in the document]
Comment (avclarke) [reply README.md]: This was a typo, should be like your first suggestion, thanks for spotting

**Comment by PaulLeCam** [comment]
@avclarke please feel free to merge whenever you want, unless you're still planning on updating some contents?

**Comment by avclarke** [comment]
@PaulLeCam I was planning on holding off until we get the launcher app creation flow merged and then updating the docs once that's in.

File: packages/cli/src/commands/app/setPermissions.js [hunk 2]
```diff
@@ -26,6 +26,11 @@ export default class SetPermissionsCommand extends Command {
       return
     }
 
+    if (!this.flags.id) {
+      this.log('Please provide an app ID (--d <APP_ID>)')
```
Comment (PaulLeCam) [inline packages/cli/src/commands/app/setPermissions.js]: Typo in the flag, it's missing the `i`

File: packages/sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Mainframe SDK
+
+Javascript SDK for communicating with the Mainframe launcher and daemon.
+
+```
+import sdk from '@mainframe/sdk'
```
Comment (PaulLeCam) [inline packages/sdk/README.md]: Should be import `MainframeSDK` here rather than `import sdk`

**Pull request merged**